aspnetcore/signalr/javascript-client.md

```diff
@@ -75,7 +75,7 @@ To prevent a malicious site from reading sensitive data from another site, [cros
 
 ## Call hub methods from the client
 
-JavaScript clients call public methods on hubs via the [invoke](/javascript/api/%40microsoft/signalr/hubconnection#invoke-string--any---) method of the [HubConnection](/javascript/api/%40microsoft/signalr/hubconnection). The `invoke` method accepts:
+JavaScript clients call public methods on hubs via the [invoke](/javascript/api/%40microsoft/signalr/hubconnection#invoke_T__T__string__any___) method of the [HubConnection](/javascript/api/%40microsoft/signalr/hubconnection). The `invoke` method accepts:
```

Comment:
```suggestion
JavaScript clients call public methods on hubs via the [invoke](/javascript/api/%40microsoft/signalr/hubconnection@microsoft-signalr-hubconnection-invoke) method of the [HubConnection](/javascript/api/%40microsoft/signalr/hubconnection). The `invoke` method accepts:
```

Reply:
pls see the review page:https://review.docs.microsoft.com/en-us/javascript/api/@microsoft/signalr/hubconnection?branch=main&view=signalr-js-latest#invoke_T__T__string__any___